packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -691,6 +693,9 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
   /// The type of shape.
   final BoxShape shape;
 
+  /// {@macro flutter.widgets.Clip}
+  final Clip clip;
```

Comment:
you also need to update the clippers, right? ClipRect, ClipRRect, etc.

Reply:
ClipRRect, ClipRect... are updated through CustomClipper<T> (which you last commented in proxy_box.dart)

Reply:
Done.